mono/mini/method-to-ir.c

```diff
@@ -7158,7 +7158,9 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 
 					g_assert (!fsig->hasthis && !fsig->pinvoke);
 
-					gboolean inline_wrapper = cfg->opt & MONO_OPT_INLINE || cfg->compile_aot;
+					/* UNITY: avoid inlining as no LMF will be setup, causing problems with exception handling. See FB case 1396617
```

Comment:
Can you add a comment in the code back to this issue?
https://github.com/dotnet/runtime/issues/64073

easier to track in the future.

Reply:
Done.